src/ua/hillel/Animals.java

```diff
@@ -1,6 +1,6 @@
 import java.util.Scanner;
 
-public class Animal {
+public class  Animal {
     private int AnimalCount;
 
     public int getAnimalCount() {
```

Comment:
Class' members should have a specific order:

constants
static variables
instance variables
constructors
getters + setters
public methods
private methods
Please change this class' members order according to the info in this comment.